projects/vdk-plugins/vdk-jupyter/vdk-jupyterlab-extension/vdk_jupyterlab_extension/vdk_ui.py

```diff
@@ -156,3 +157,57 @@ def create_deployment(
             enabled=enabled,
         )
         return output
+
+    @staticmethod
+    def get_notebook_info(cell_id: str, pr_path: str):
+        """
+        Return information about the notebook that includes a cell with a given id
+        :param cell_id: the id of the cell
+        :param pr_path: the path to the parent directory of the notebook
+        :return: path of the notebook, index of the cell
+             if the specified parent directory does not exist, an empty dictionary is returned.
+        """
+        if not os.path.exists(pr_path):
+            pr_path = os.getcwd() + pr_path
+            if not os.path.exists(pr_path):
+                return {"path": "", "cellIndex": ""}
+        notebook_files = [
+            x
+            for x in pathlib.Path(pr_path).iterdir()
+            if (x.name.lower().endswith(".ipynb"))
+        ]
+        for notebook_file in notebook_files:
+            with open(notebook_file) as f:
+                notebook_json = json.load(f)
+                cell_index = 0
+                for jupyter_cell in notebook_json["cells"]:
```

Comment:
Nit: Consider using `enumerate` here and getting rid of the cell_index variable.

Reply:
ready